docs/topic-guides/training-units.txt

```diff
@@ -0,0 +1,180 @@
+.. _topic-guides_training-units:
+
+################
+ Training Units
+################
+
+With the flexibility of Determined’s experiment configuration, the combinations of possible
+configuration settings available are numerous. This guide aims to help you better understand the
+options available for training units to make the right choice for your model.
+
+Determined supports configuring the following fields:
+
+-  ``searcher.max_length`` in most searchers
+
+-  ``searcher.length_per_round`` in PBT
+
+-  ``min_validation_period``
+
+-  ``min_checkpoint_period``
+
+-  ``searcher.budget`` when using ``adaptive_simple``
+
+in the following units:
+
+-  ``records``: A *record* is a single labeled example (sometimes called a sample).
+
+-  ``batches``: A *batch* is a group of records (the number of records in a batch is configured via the ``global_batch_size`` hyperparameter).
+
+-  ``epochs``: An *epoch* is typically single copy of the entire training data set.
+
+
+The sections below present different configurations that result in the same effective training
+length and behavior but use different training units.
+
+In general, conversions like the conversions between the configurations below will result
+in identical behavior except in the following cases:
+
+-  When a quantity is too granular to be converted to a less granular unit. For example, since
+   partial values aren't accepted, moving from 50 ``records`` and to ``epochs`` with 100 records per
+   epoch isn't possible. 
+
+-  When using a variable hyperparameter ``global_batch_size``. These configurations cannot be
+   expressed correctly in terms of batches, since expressing the training length in terms of batches
+   will result in a variable number of records.
+
+-  When using an adaptive searcher such as ``adaptive_simple``. When the length of each rung is
```

Comment:
We should prefer talking about `adaptive_asha`.

Reply:
The way they (old and new asha) behave with the atomicity is different, so I tried to write something more clear that applies to both.